src/tokens.js

```diff
@@ -1132,6 +1132,103 @@ const tokens = {
     return uid === HATHOR_TOKEN_CONFIG.uid;
   },
 
+  /**
+   * Checks if this transaction is the creation of an NFT following the NFT Standard Creation.
+   * @see https://github.com/HathorNetwork/rfcs/blob/master/text/0032-nft-standard.md#transaction-standard
+   * @param {Object} tx Transaction with decoded inputs and outputs
```

Comment:
I think we should try to use here the `Transaction` model object, instead of an object

Reply:
A full refactor was made to move this code into the `Transaction` class on [4c5e1b9](https://github.com/HathorNetwork/hathor-wallet-lib/pull/378/commits/4c5e1b9adad391aa3e39087ecbf6c7df01c00ef7). I think it is now better to understand the context in which to use this function.